docs/concepts/ipfs-implementations.md

```diff
@@ -12,26 +12,33 @@ Looking into implementing IPFS yourself?
 | Kubo             | <https://github.com/ipfs/kubo>                                          | go                     | Generalist daemon oriented IPFS implementation with an extensive HTTP API.                                               |
-| Linux2ipfs       | <https://github.com/Jorropo/linux2ipfs>                                 | go                     | Small pipeline and extreme-performance oriented implementation to upload files and deltas to pinning services very fast. |
 | Lotus            | <https://github.com/filecoin-project/lotus>                             | go                     | Filecoin node handling consensus, storage providing, making storage deals, importing data, ...                           |
-| py-ipfs          | <https://github.com/ipfs-shipyard/py-ipfs>                              | python                 | Python IPFS implementation.                                                                                              |
-| rust-ipfs        | <https://github.com/rs-ipfs/rust-ipfs>                                  | rust                   | Rust IPFS implementation.                                                                                                |
 | whypfs           | <https://github.com/whyrusleeping/whypfs>                               | go                     | Daemon based on the same building blocks as Kubo but with some options tweaking for more performance.                    |
+
+## Legacy
+
+| Name             | URL                                                                     | Language(s)            | What it's trying to do                                                                                                   |
+|------------------|-------------------------------------------------------------------------|------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| auspinner        | <https://github.com/2color/auspinner>                                   | go                     | CLI tool to deal with the pinning service API and upload files through bitswap.                                          |
+| Agregore         | <https://github.com/AgregoreWeb/agregore-ipfs-daemon>                   | go, javascript         | Mobile friendly Kubo daemon.                                                                                             |
+| ipfs tiny        | <https://gitlab.com/librespacefoundation/ipfs-tiny>                     | c++                    | Tiny embeddable, os-independent IPFS implementation.                                                                     |
+| ipget            | <https://github.com/ipfs/ipget>                                         | go                     | Minimal wget insipired tool to download files from IPFS nodes over bitswap.                                              |
```

Comment:
Line 41, spelling flagged by vale

`insipired`

Reply:
Wow don't know how that slipped by, let's leave it as a legacy typo.

Haha jk, I'll make the fix.

Reply:
"legacy typo" 😆